webapp/ordering/models.py

```diff
@@ -364,6 +392,12 @@ def send_distribution_mails(self):
                 )
                 mail_user(user, *rendered_template_vars)
 
+    @property
+    def get_pickup_location(self):
```

Comment:
conventie is om óf een functie te hebben die `get_something` heet óf een property `something` :)